packages/arb-token-bridge-ui/src/util/TokenUtils.ts

```diff
@@ -35,6 +41,29 @@ export type Erc20Data = {
   address: string
 }
 
+export function isERC20BridgeToken(
+  token: Token | NativeCurrencyErc20
+): token is ERC20BridgeToken {
+  if (!token) {
+    return false
+  }
+  return typeof (token as ERC20BridgeToken).address !== 'undefined'
+}
+
+export function isEther(
```

Comment:
This could be confusing because if `token` is null then it could still be ether, but this method would then return false.

I'm open to renaming it if it's too confusing